index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta name="description" content="Erol Kucuk Microverse Portfolio">
+  <title>Microverse Erol Portfolio</title>
+  <link href="https://fonts.googleapis.com/css2?family=**Quicksand:wght@500**&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+
+</head>
+
+<body>
+  <!-- Header -->
+  <header class="header">
+    <div class="leftsection" href="#"><img src="images/My Logo.png"></a></div>
+     <nav class="rightsection">
+      <a class="iconmenu" href="#"><img src="images/Enabled.png"></a>
+     </nav>
+  </header>
+  <!-- End Header -->
+
+  <!-- Introduction Area -->
+  <main class="headline">
+    <h1 class="introduction"> i'm Erol Glad to see you!</h1>
+    <p class="intro-description">
+      Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my
+      works.
+      If you like what you see and have a project you need coded, do not hesitate and contact me.
+    </p>
+    <!-- End Introduction Area -->
+    <p class="support">LET'S CONNECT</p>
+    <ul class="social-media">
+      <li><a href="#"><img src="images/Vector (1).svg" alt=""></a></li>
+      <li><a href="#"><img src="images/Vector (2).svg" alt=""></a></li>
+      <li><a href="#"><img src="images/Vector (3).svg" alt=""></a></li>
+
+     </ul>
+     </main>
```

Comment:
- [ ] As the previous reviewer requested, please stick as much as possible to the Figma template 🎨. At the moment, your project has some visual mismatches with it, as shown below:

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/205364047-5c2feac6-be2c-4688-8817-45b677d9802c.png) | ![image](https://user-images.githubusercontent.com/77069011/205364005-767ef117-ca6c-486a-b5b6-1526603cd023.png) |

Please consider fixing the font, the spacing, and the background image. 🚧 

One of the main goals of this project is that you demonstrate the capability of following as closely as possible a given design, please consider implementing these changes! 🤓 🛠️